tests/phpunit/WikiManagerTest.php

```diff
@@ -1,5 +1,10 @@
 <?php
```

Comment:
I believe that `@coversDefaultClass` needs to be the full namespaced class name.